ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/ManagementPageModelTests.cs

```diff
@@ -75,6 +75,7 @@ public async Task WhenOnGetAsync_MissingCaseUrn_ReturnsPageModel()
 			Assert.That(pageModel.CaseModel.StatusName, Is.EqualTo(caseModel.StatusName));
 			Assert.That(pageModel.CaseModel.TrustName, Is.EqualTo(caseModel.TrustName));
 			Assert.That(pageModel.CaseModel.TrustNameTitle, Is.EqualTo(caseModel.TrustName.ToTitle()));
+			Assert.That(pageModel.CaseModel.TrustDetailsModel, Is.EqualTo(caseModel.TrustDetailsModel));
```

Comment:
If we have more tests around TrustDetailsModel further in the PR ignore the comment. I would add a few tests internally to the other models.